webproject/src/main/java/com/google/sps/dataManagers/UserRoomManager.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.dataManagers;
+
+import java.util.Optional;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.ValueEventListener;
+import com.google.sps.data.UserRoom;
+import com.google.sps.util.FirebaseUtil;
+
+@Singleton
+public class UserRoomManager {
+    private FirebaseUtil firebaseUtil;
+
+    @Inject
+    public UserRoomManager(FirebaseUtil firebaseUtil) {
+        this.firebaseUtil = firebaseUtil;
+    }
+
+    public void addUserRoom(String userEmail, String roomId) {
```

Comment:
I think this function should return the UserRoom object added

Reply:
done